cumulusci/cli/ui.py

```diff
@@ -89,27 +89,29 @@ def echo(self, plain=False):
         Automatically falls back to AsciiTable if there's an encoding error.
         """
         if plain or os.environ.get("TERM") == "dumb":
-            self.ascii_table()
-            return None
+            table = self.ascii_table()
+        else:
+            table = self.table
+        click.echo(table)
 
+    @property
+    def table(self):
```

Comment:
What about `__str__`?

Reply:
Huh, forgot about `__str__`. Instead of this, or in addition to it?

Reply:
Instead of. I think `str(table)` is a clearer API than `table.table`

Reply:
Well, when you write it out like that... :smirk:

I'll update.